tutorials/s4sdk-odata-service-cloud-foundry/s4sdk-odata-service-cloud-foundry.md

```diff
@@ -258,7 +258,7 @@ $destinations='[{name: "MyErpSystem", url: "https://URL"}]'
 **Note:** Some services, other than the sandbox service and the mock server, may require user credentials for the request execution.
```

Comment:
Wrong format. Do not use the label Note.

https://workzone.one.int.sap/site#workzone-home&/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true